client/app/catalog/catalog.html

```diff
@@ -34,10 +34,9 @@
 
   <!-- Start photo catalog -->
   <div id="catalog-photos" class="col-sm-10">
-    <div class="row">
 
       <!-- Product info -->
```

Comment:
This should be inside an ng-repeat. You should not be repeating html elements!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

It causes more merge conflicts, and makes it much harder to merge in changes to logic.